communication-protocols/LoRaWAN/config.json

```diff
@@ -1,8 +1,92 @@
 {
+    "identifier": "devEUI",
     "fields": [
-        {"label":"Dev EUI", "value":"devEUI", "isRequired": true},
-        {"label":"App EUI", "value":"appEUI", "isRequired": true},
-        {"label":"App Key", "value":"appKey", "isRequired": true}
-    ],
-    "identifier": "devEUI"
-}
+        {
+            "label": "Dev EUI",
+            "value": "devEUI",
+            "isRequired": true,
+            "canBeEdited": false,
+            "validationRules": [
+                {
+                    "can": true,

```

Comment:
What is "can" supposed to mean ? isn't it "required on https://ant.design/components/form#rule ?

Reply:
oops that's a mistake